### PR TITLE
Add documentation templating

### PR DESCRIPTION
**Pull request by fhennig**
https://github.com/stackabletech/documentation/issues/237

### CODE REVIEW

File: template/docs/modules/getting_started/pages/install_the_operator.adoc.j2 [hunk 1]
```diff
@@ -0,0 +1,43 @@
+= Installation
+
+There are 2 ways to run the {[ operator.pretty_string }] Operator:
+
+1. Using xref:stackablectl::index.adoc[]
+
+1. Using Helm
```
Comment (maltesander) [inline template/docs/modules/getting_started/pages/install_the_operator.adoc.j2:7]:
```suggestion
2. Using Helm
```

**Review by maltesander — COMMENTED** [review]
One minor thing

**Review by maltesander — APPROVED** [review]
LGTM. We discussed and merge as is. This requires all tools affected by templating to have the `installation.adoc` manually removed as well as the the `nav.adoc` updated (in the resulting templating PRs).

**Comment by fhennig** [comment]
We're not doing it, there's too much different stuff in there it can't be templated

**Pull request closed without merging**